### PR TITLE
Privilege fixes on rkt+Newton

### PR DESCRIPTION
**Pull request by Quentin-M**

### CODE REVIEW

File: horizon/manifest.jsonnet [hunk 1]
```diff
@@ -27,7 +27,7 @@ kpm.package({
 
     network: {
       external_ips: [],
-      port: 80,
+      port: 8080,
```
Comment (DTadrzak) [inline horizon/manifest.jsonnet]: Apache2 has extended linux_capabilities about cap_net_bind_service. So you don't need to do it even if you start as a non-root.

File: cinder/templates/volume.yaml.j2 [hunk 2]
```diff
@@ -9,6 +9,12 @@ spec:
     spec:
       nodeSelector:
         {{ deployment.control_node_label}}: enabled
+      {% if deployment.engine == "rkt" -%}
+      # Required by rkt to let nova-api (metadata service) execute:
```
Comment (DTadrzak) [inline cinder/templates/volume.yaml.j2]: it is cinder-volume :)

File: horizon/manifest.jsonnet (hunk 1 above)
Comment (Quentin-M) [inline horizon/manifest.jsonnet]: Right, for some reason, with the Newton image:
```
+ apache2 -DFOREGROUND
(13)Permission denied: AH00072: make_sock: could not bind to address 0.0.0.0:80
no listening sockets available, shutting down
AH00015: Unable to open logs
```

Trying to figure out why now.

**Comment by ss7pro** [comment]
LGTM

**Pull request merged**